src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1393,6 +1393,27 @@ public void LanguageVersion_TryParseDisplayString(string input, bool success, La
         }
 
         [Fact]
+        public void LanguageVersion_CommandLineUsage()
+        {
+            var expected = Enum.GetValues(typeof(LanguageVersion)).Cast<LanguageVersion>().Select(v => v.ToDisplayString());
+            string help = CSharpResources.IDS_CSCHelp;
+
+            var rangeStart = help.IndexOf("/langversion");
+            var rangeEnd = help.IndexOf("/delaysign");
+            Assert.True(rangeEnd > rangeStart);
+
+            string helpRange = help.Substring(rangeStart, rangeEnd - rangeStart).ToLowerInvariant();
+
+            foreach (var version in expected)
+            {
+                Assert.True(helpRange.Contains(version), $"Missing version '{version}'");
```

Comment:
>Assert.True(helpRange.Contains(version), $"Missing version '{version}'"); [](start = 16, length = 73)

It doesn't look like the check is strong enough. For example, it looks like it will find "7" in "7.1" and will not fail when "7" is not there.